src/openzaak/components/besluiten/tests/admin/test_besluitinformatieobject_audittrail.py

```diff
@@ -15,6 +15,7 @@
 from ..utils import get_operation_url
 
 
+@override_settings(CMIS_ENABLED=False)
```

Comment:
Please remove `CMIS_ENABLED=False` everywhere (and override_settings import if not used)